### ISSUE TITLE
Ghost empty when used in a table

### ISSUE THREAD

**Issue by rogerfar** [lukasoppermann/html5sortable#190]
See this fiddle: http://jsfiddle.net/qjdunLnu/

When you drag the numbers, they are just empty. Couldn't figure out if it is an issue with the library or with the browser.

**Comment by yesman85** [comment]
Guess this project is dead.

**Comment by yesman85** [comment]
Guess this project is dead.

**Comment by lukasoppermann** [comment]
Hey, support for tables was never really a goal. If you want to send a PR to fix this, I would be very happy to merge it.

**Comment by ninze** [comment]
This seems to be a browser issue: works well with firefox. In webkit browsers, some table cells are displayed and some aren't. Trying to figure out how webkit decides what to display/hide.

**Comment by lukasoppermann** [comment]
Hmm, interesting. Please feel free to submit a PR if you figure out how ti solve it. Thank you.

**Comment by ninze** [comment]
Safari will display contents of the TD cell as long as there are no inline elements inside the TD. For example `<td>text</td>` will be displayed, but not `<td>text<i></i></td>`.

Chrome will not display anything in either case.

The problem seems to be with the `display: table-row` value. If we set display of UL to `table`, and display of LI to `table-row`, and inner divs as `table-cell`, then the same behavior happens as with regular tables. Generally, the sortable items should not have `display:table-row` set.

I have looked through the code and I don't think there is anything wrong with it, at least the part where the ghost is being created and set. In my case I was able to replace the traditional table with table layout not using display: table-row.
